cmd/server/main.go

```diff
@@ -47,36 +44,15 @@ func run(cfg *config.ServerConfig) error {
 	repos := repositories.NewRepository(store)
 	h := handlers.NewHandler(server.NewMetricsService(repos, cfg.Log), cfg.Log)
 
-	cfg.Log.DebugContext(context.Background(), "running server 🔥 on port: "+cfg.Address)
+	cfg.Log.DebugContext(context.Background(), "running server 🔥 on port: "+cfg.Envs.Address)
```

Comment:
It's better to do it like this, as it corresponds more to the  principles of structural logging

```suggestion
	cfg.Log.DebugContext(context.Background(), "running server 🔥", slog.String("address", cfg.Envs.Address))
```